src/components/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { NO_DATA } from '../../utils/constants';
+
+type Props = {
+  person: Person;
+  selectedPersonSlug: string;
+};
+
+export const TableRow: React.FC<Props> = ({ person, selectedPersonSlug }) => {
+  const {
+    sex,
+    born,
+    died,
+    slug,
+    mother,
+    father,
+    motherName,
+    fatherName,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === selectedPersonSlug,
+      })}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother ? (
+          <PersonLink person={mother} />
+        ) : (
+          motherName || NO_DATA
+        )}
+      </td>
```

Comment:
No need to re-write, just an idea for the future

You can add a property `personNameFallback` (optional) to `PersonLink` and make the property person optional, so the PersonLink itself would be able to render a plain person name, `NO_DATA` or person link, depending on the information provided.

```suggestion
      <td>
          <PersonLink person={mother} personNameFallback={motherName} />
      </td>
```